test/OWNERS

```diff
@@ -28,6 +28,7 @@ reviewers:
   - vishh
 approvers:
   - bowei # for test/e2e/{dns*,network}.go
+  - bsalamat # for test/e2e/scheduling
```

Comment:
Please also add yourself to reviewers.

Please consider moving this to `test/e2e/OWNERS` or `test/e2e/scheduling/OWNERS`